DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -380,6 +384,35 @@ public void Error_when_registering_delegate_fails_fast()
             ).MustNotHaveHappened();
         }
 
+        [Test]
+        public void Registering_delegate_calls_data_service_to_set_prn()
+        {
+            // Given
+            const string clientIp = ApprovedIpPrefix + ".100";
+            const string prn = "PRN";
+            var model = RegistrationModelTestHelper.GetDefaultDelegateRegistrationModel();
+
+            // When
+            var (_, approved) = registrationService.RegisterDelegate(
+                model,
+                clientIp,
+                false,
+                prn
+            );
+
+            // Then
+            A.CallTo(
+                    () =>
+                        userDataService.UpdateDelegateProfessionalRegistrationNumber(
+                            A<int>._,
+                            prn,
+                            true
+                        )
+                )
+                .MustHaveHappenedOnceExactly();
+            Assert.That(approved);

```

Comment:
To avoid mixing convetions could this be changed to the `FluentAssertions` way -> `approved.Should().BeTrue()`.
Although I note this has been used elsewhere in this class but the more common `FluentAssertions` is also here in this class too.

Reply:
Yeah I just copied it from the other test in the file, I've changed both now.

Reply:
No worries, thanks for changing both!